ci/kokoro/windows/build-dependencies.ps1

```diff
@@ -110,6 +110,8 @@ foreach ($pkg in $packages) {
     if ($LastExitCode) {
         throw "vcpkg install $pkg failed with exit code $LastExitCode"
     }
+    # Remove the buildtree directory
+    Remove-Item -path t:\src\github\vcpkg\buildtrees -recurse
```

Comment:
FYI: Greg's gone for the "rest of the year", so I'm happy to review.

In the meantime, I guess you saw ...
```
Remove-Item : Cannot find path 'T:\src\github\vcpkg\buildtrees' because it does not exist.
At T:\src\github\google-cloud-cpp-spanner\ci\kokoro\windows\build-dependencies.ps1:114 char:5
+     Remove-Item -path t:\src\github\vcpkg\buildtrees -recurse
```

Reply:
Thanks! I think I fixed it.